qadence/backends/adjoint.py

```diff
@@ -105,27 +106,28 @@ def _apply_adjoint(ctx: Any, op: Module) -> list:
                 ctx.projected_state = apply_operator(
                     ctx.projected_state, op.dagger(values), op.qubit_support
                 )
-            elif isinstance(op, ScalePyQOperation):
-                ctx.out_state = apply_operator(ctx.out_state, op.dagger(values), op.qubit_support)
+            elif isinstance(op, pyq.Scale):
```

Comment:
Couldn't you simply import them ? I guess there is no ambiguity.

Reply:
file removed